streams/src/test/java/org/apache/kafka/test/MockInternalProcessorContext.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.kafka.test;
+
+import org.apache.kafka.streams.processor.MockProcessorContext;
+import org.apache.kafka.streams.processor.internals.InternalProcessorContext;
+import org.apache.kafka.streams.processor.internals.ProcessorNode;
+import org.apache.kafka.streams.processor.internals.ProcessorRecordContext;
+import org.apache.kafka.streams.processor.internals.metrics.StreamsMetricsImpl;
+import org.apache.kafka.streams.state.internals.ThreadCache;
+
+public class MockInternalProcessorContext extends MockProcessorContext implements InternalProcessorContext {
```

Comment:
`MockProcessorContext` is pretty nice for unit tests, but we need `InternalProcessorContext` for the suppress processor.